h2o-algos/src/main/java/hex/ensemble/Metalearner.java

```diff
@@ -47,7 +48,8 @@ void init(Frame levelOneTrainingFrame,
             Job metalearnerJob,
             StackedEnsembleParameters parms,
             boolean hasMetalearnerParams,
-            long metalearnerSeed) {
+            long metalearnerSeed,
+            long remainingTime) {
```

Comment:
just to bother you:
classes should look like they are self-contained (even when it's not the case), but from a pure semantic perspective `remainingTime` systematically raises the question "remaining of what?".
The answer is simple: "remaining time from the caller perspective".
So, when naming things, we should try to forget (temporarily) the specific conditions in which this thing is being used: what is "remaining time" from SE point of view is just "max runtime" from metalearner perspective.

To your defense: almost all parameters in this class are badly named... 3 different case conventions, redundant use of word `metalearner` in a class named `Metalearner`... even the name `levelOne`... what if I want to do multi-layer stacking?

Reply:
No bother at all. I really appreciate these kind of remarks. I renamed it to `maxRuntimeSecs`  that should be clearer, I hope.